green/reporting.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import unicode_literals
```

Comment:
This file is all about JUnit, used when you specify an option called `junit_report`.  Let's go ahead and name this file `junit.py` for consistency.  I'm a big fan of using the same name for the same things.  (and `test/test_reporting.py` would then become `test/test_junit.py` for the same reason).

Reply:
Done.